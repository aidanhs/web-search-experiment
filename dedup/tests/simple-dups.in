<DOC>
<DOCNO>1</DOCNO>
<TEXT>
hello how are you
</TEXT>
</DOC>

<DOC>
<DOCNO>2</DOCNO>
<TEXT>
hello how are you
</TEXT>
</DOC>

<DOC>
<DOCNO>3</DOCNO>
<TEXT>
hello how are you
</TEXT>
</DOC>

<DOC>
<DOCNO>4</DOCNO>
<TEXT>
hello how are you
</TEXT>
</DOC>

<DOC>
<DOCNO>5</DOCNO>
<TEXT>
hello how are you
</TEXT>
</DOC>

<DOC>
<DOCNO>6</DOCNO>
<TEXT>
hello how are you
</TEXT>
</DOC>

<DOC>
<DOCNO>7</DOCNO>
<TEXT>
hello how are you
</TEXT>
</DOC>

<DOC>
<DOCNO>8</DOCNO>
<TEXT>
hello how are you
</TEXT>
</DOC>

<DOC>
<DOCNO>9</DOCNO>
<TEXT>
hello how are you
</TEXT>
</DOC>

<DOC>
<DOCNO>10</DOCNO>
<TEXT>
hello how are you
</TEXT>
</DOC>
