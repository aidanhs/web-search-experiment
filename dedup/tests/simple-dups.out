2, 1, 9, 4, 10, 3, 6, 5, 8, 7
